src/Main.java

```diff
@@ -0,0 +1,45 @@
+// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
+// then press Enter. You can now see whitespace characters in your code.
+public class Main {
+    public static void main(String[] args) {
+        // Press Opt+Enter with your caret at the highlighted text to see how
+        // IntelliJ IDEA suggests fixing it.
+        System.out.printf("Hello and welcome!");
```

Comment:
printf en println zijn twee verschillende methodes. Een van de belangrijkste verschillen zit in het feit dat println een "newline character" aan het eind van de tekst print. Bij printf moet je die zelf nog toevoegen ("\n"). 

Een ander verschil is dat printf meerdere parameters kan ontvangen. De eerste parameter is een format string. Deze heb je  nu ook al meegegeven en wordt dus gewoon geprint. Als je zogenaamde "format specifiers" gebruikt (er zijn er een aantal van, google er maar eens op), kun je als aanvullende parameters de invulling van die specifiers meegeven. Bijvoorbeeld: 

`System.out.printf("Hallo %s, dit is een voorbeeld", "Arjen");` 

Dit print "Hallo Arjen, dit is een voorbeeld"

Je kunt meerdere specifiers gebruiken. Elke nieuwe parameter is de betekenis van de volgende specifier. Let er op dat specifiers voor specifieke types zijn. Zo is %s enkel voor strings en %i voor integers.